srcpkgs/nody-greeter/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'nody-greeter'
+pkgname=nody-greeter
+version=1.3.1
+revision=1
+archs="i686 x86_64"
+build_style=meta
+bootstrap=true
+conf_files="/etc/lightdm/web-greeter.yml /etc/lightdm/Xgreeter /etc/xdg/lightdm/lightdm.conf.d/90-greeter-wrapper.conf"
+hostmakedepends="base-devel nodejs python3"
+makedepends="base-devel libxcb-devel libX11-devel liblightdm-gobject gobject-introspection cairo"
+depends="cairo libX11 libxcb lightdm liblightdm-gobject"
```

Comment:
depends= are mostly generated automatically, see `04-generate-runtime-deps` in build log.